achievements/labelBabyJunior.achievement.js

```diff
@@ -0,0 +1,22 @@
+var labelBabyJunior = {
+	name: 'Label Baby Junior',
+	check: function(pullRequest, shall) {
+		if (isManyLabels(pullRequest))  {
+			var achievement = {
+				avatar : 'images/achievements/theLabelMaker.achievment.jpg',
+				name: 'The Label Maker',
+				short: 'Is this a label maker?',
+				description: 'You\'ve put many labels, thank you for organizng. You\'re a gift that keeps on re-giving' ,
+				relatedPullRequest: pullRequest._id
+			};
+
+			shall.grant(pullRequest.creator.username, achievement);
+		}
+	}
+};
+
+function isManyLabels(pullRequest) {
+	var labels = pullRequest.labels;
+	return labels && labels.length > 3;
```

Comment:
I think that 4 labels as a minimum is a bit low.
In Kibibit-Code-editor we have at least 4 PRs with 4 labels each. 
maybe we should use `labels.length > 5` ?
and Kibibit is a small organization with a small team.

Take a look at [this PR](https://github.com/angular/angular/pull/14039) from AngularJS (Google's Javascript framework) - it has 8(!) labels. They have tons of PRs with 4-5 labels.
LMKWYT